cms/src/components/content/EditContentParentValidation.vue

```diff
@@ -142,6 +143,21 @@ watch(
 <template>
     <div class="rounded-md bg-zinc-100 p-3 shadow-inner">
         <div class="flex flex-col gap-2">
+            <div v-if="!parentIsValid" class="rounded-md bg-zinc-50 p-4 shadow">
```

Comment:
```suggestion
            <div v-if="!parentIsValid" class="rounded-md bg-zinc-50 p-4 mb-2 shadow">
```
add a little bottom margin?